### ISSUE TITLE
Discussing the code and potential improvements 

### ISSUE THREAD

**Issue by atulag0711** [KislayaRavi/scout-Nd#1]

**Comment by atulag0711** [comment]
- do we haveto use both jax.scipy norm and numpyro dist?
- Now the starting values are set to vector of 1st as mean and 0.1 of that as sd. We need to be able to input it no, or always normalize the input?
- The funtion call seem to vectorized. Will it be the case always? We can discuss this now and extend this. 
- have doubts about how the constraints are implemented but will get back (note for self)

**Comment by atulag0711** [comment]
also what is the best way that we both have same conda env. adding env.yaml file in the repo?
